package.json

```diff
@@ -85,6 +85,14 @@
           "group": "inline/cell@2",
           "category": "Gather"
         }
+      ],
+      "interactive/cell/title": [
+        {
+          "command": "gather.gatherCodeNativeNotebook",
+          "when": "notebookType == interactive && notebookCellType == code",
```

Comment:
Does this action and the action above for notebooks need to be gated on Python language in the cell?

Reply:
It checks in code, but yeah, this will be added as part of #57